noxfile.py

```diff
@@ -22,4 +22,6 @@ def lint(session):
 @nox.session
 def tests(session):
     session.install(".[test]")
-    session.run("pytest", "-v")
+    session.run("pytest", "-v", "tests/test_read_pdf_table.py")
+    session.run("pytest", "-v", "tests/test_read_pdf_jar_path.py")
```

Comment:
This is due to jpype limitation that not allow shutdown JVM. Hence, we need to make separate session run for JVM option testing.

Reply:
> This is due to jpype limitation that doesn not allow shutdown JVM

how about `jpype.shutdownJVM()`?

Reply:
According to the document, we have to reboot Python to relaunch JVM.

https://jpype.readthedocs.io/en/latest/api.html#jpype.shutdownJVM
> Due to limitations in the JPype, it is not possible to restart the JVM after being terminated.

Reply:
Ah, I see.